jaxrs/server-adapters/resteasy-netty4/src/test/java/org/jboss/resteasy/test/NettyTest.java

```diff
@@ -190,4 +197,36 @@ public void testPost() {
       String result = (String) target.request().post(Entity.text(postBody), String.class);
       Assert.assertEquals(postBody, result);
     }
+
+
+   /**
+    * Per the HTTP spec, we must allow requests like:
+    *
+    * <pre>
+    *     GET http://www.example.com/content HTTP/1.1
+    *     Host: www.example.com
+    * </pre>
+    *
+    * <blockquote>
+    * RFC 2616 5.1.12:
+    * To allow for transition to absoluteURIs in all requests in future
+    versions of HTTP, all HTTP/1.1 servers MUST accept the absoluteURI
+    form in requests, even though HTTP/1.1 clients will only generate
+    them in requests to proxies.
+    </blockquote>
+    * @throws Exception
+    */
+   @Test
+   public void testAbsoluteURI() throws Exception {
+      String uri = generateURL("/test");
+
+      Socket client = new Socket(getHost(), getPort());

```

Comment:
This tests uses sockets directly since the Resteasy client builders don't support making requests with absolute URIs in the request line.